modules/gdscript/gdscript_warning.cpp

```diff
@@ -155,6 +155,10 @@ String GDScriptWarning::get_message() const {
 		case INT_ASSIGNED_TO_ENUM: {
 			return "Integer used when an enum value is expected. If this is intended cast the integer to the enum type.";
 		}
+		case STATIC_CALLED_ON_INSTANCE: {
+			CHECK_SYMBOLS(2);
+			return "The function '" + symbols[0] + "()' is a static function but was called from an instance. Instead, it should be directly called from the type: '" + symbols[1] + "." + symbols[0] + "()'.";
```

Comment:
Could use `vformat` and R-strings, see examples above.

Reply:
Done